spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
It's definitely better to have test than not. Before was worse.

I don't care so much about the profiling actually since the test suite takes a seconds to run. For bigger test suites, I would feel much stronger to leave the profiling as an opt-in. So you can leave that in if you'd like :)